lib/api.ml

```diff
@@ -7,7 +7,7 @@ module type Github = sig
   val get_api_commit : ctx:Context.t -> repo:repository -> sha:string -> (api_commit, string) Result.t Lwt.t
   val get_pull_request : ctx:Context.t -> repo:repository -> number:int -> (pull_request, string) Result.t Lwt.t
   val get_issue : ctx:Context.t -> repo:repository -> number:int -> (issue, string) Result.t Lwt.t
-  val get_compare : ctx:Context.t -> repo:repository -> basehead:Github.basehead -> (compare, string) Result.t Lwt.t
+  val get_compare : ctx:Context.t -> repo:repository -> basehead:string * string -> (compare, string) Result.t Lwt.t
```

Comment:
where is this change coming from?

Reply:
This is actually the same type, I just used Api inside the github module (`~(api : (module Api.Buildkite))`), so a reference here would be a circular dependency. If I don't pass the whole module around then maybe this will change.